README.md

```diff
@@ -1 +1,56 @@
-# solution-pack-malware-response-using-symantec-solutions
+# Malware Response Using Symantec Solutions
+
+## Overview
+### Introduction
+This article describes the  Malware Response Using Symantec Solutions Pack. The solution pack demonstrates the scenarios and use cases around Malware resported by Symantec EDR.
+
+The scenario demonstrates and generates a demo alert for the Alert Type 'Malwarel' reported by Symantec EDR.
+
+The usecase deals with containment and investigation procedures once the Symantec EDR reports a Malware Alert.
+
+    Step 1: Get asset details from Symantec EDR
+    Step 2: Get File Reputation using Symantec DeepSight Intelligence
+    Step 3: If File Reputation is Malicious then ask Analyst for Endpoint Isolation 
+    Step 4: Delete the malicious file hashes from the device using Symantec ATP 
+    Step 5: Isolate the device using Symantec ATP 
+    Step 6: Block the User in Symantec CloudSOC
+    Step 7: Send an email to the IT team that the asset has been isolated due to malware infection and ask for an Initiate asset clean-up procedure.
+    Step 8: Close the Alert 
+
+### Usage 
+More information about usage of ”Malware Response Using Symantec Solutions” Solution Pack [here](https://github.com/fortinet-fortisoar/solution-pack-malware-response-using-symantec-solutions/blob/develop/docs/solution-pack-guide.md).
+
+## Version Information
+- Solution Pack Version: 1.0.0
+- FortiSOAR™ Version Tested on: 7.2.0
+- Authored By: Fortinet
+- Certified: No
+
+## Prerequisites
+**Solution Pack Name**|**Purpose**|**Doc Link**|
+| :- | :- | :- |
+|SOAR Framework 1.0.0|Require for Incident Response modules|[Click here](https://github.com/fortinet-fortisoar/solution-pack-soar-framework/blob/develop/README.md)|
+|SOC Simulator 1.0.1|Require for Scenario Module and SOC Simulator connector| [Click here](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/README.md)|
+
+## Contents
+1. Global Variable(s)
+    - Demo_mode: This Global variable is used to simulate the usecase using the Mock Output.
+1. Record Set(s)
+    - Scenario: Malware Response Using Symantec Solutions
+2. Connector(s)
+    **SN**|**Connector Name**|
+    | :- | :- |
+    |1|SMTP|
+    |2|Symantec CloudSOC|
```

Comment:
Symantec ATP is not mentioned here, why?

Reply:
ATP is what the software was called up until version 4.0 when we changed the name to Endpoint Detection and Response.